lib/evse_security.cpp

```diff
@@ -211,11 +216,13 @@ DeleteCertificateResult EvseSecurity::delete_certificate(const CertificateHashDa
     bool found_certificate = false;
     bool failed_to_write = false;
 
+    // TODO (ioan): load all the bundles since if it's the V2G root in that case we might have to delete
+    // whole hierarchies
     for (auto const& [certificate_type, ca_bundle_path] : ca_bundle_path_map) {
         try {
             X509CertificateBundle ca_bundle(ca_bundle_path, EncodingFormat::PEM);
```

Comment:
We should take care to be consistent with the required cert format throughout the library. This method will only see PEM certificates, so it wouldn't work correctly if there's DER certificates which other methods can see.